com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/PathTracing/Shaders/PathTracingBSDF.hlsl

```diff
@@ -387,6 +461,25 @@ bool SampleDelta(MaterialData mtlData,
     return any(outgoingDir);
 }
 
+bool SampleLambert(MaterialData mtlData,
+                   float3 inputSample,
+               out float3 outgoingDir,
+               out float3 value,
+               out float pdf)
+{
+    bool retVal = BRDF::SampleLambert(mtlData, inputSample, outgoingDir, value, pdf);
+    outgoingDir = -outgoingDir;
```

Comment:
sorry I was not asking for why the -1, I was asking why we are flipping the direction here?

Reply:
Oh right... XD
It's a back diffuse lobe, so you generate a Lambert BRDF direction as usual, and then you just flip it to the lower hemisphere. Because the lobe is azimuthally invariant, all that matters is that they share the same dot product with the normal, so that they have same pdf and value. If we were generating it in the tangent frame, we could just flip the z coord, but because it is directly in world space we flip the whole vector.